backend/inspirehep/config.py

```diff
@@ -133,7 +133,7 @@
 # JSONSchemas
 # ===========
 #: Hostname used in URLs for local JSONSchemas.
-JSONSCHEMAS_HOST = "localhost:5000"
+JSONSCHEMAS_HOST = "localhost:8000"
```

Comment:
Is this needed?

Reply:
just for consinstency with the ``SERVER_NAME``